### PR TITLE
Edit task 

### PR DESCRIPTION
**Pull request by krismutalapova**
Methods for editing task name, date, project and status have been added.
Method to remove the task has been added

### CODE REVIEW

File: src/resources/task.csv [hunk 1]
```diff
@@ -1,2 +1,6 @@
-milk,2019-12-12,(2) SHOPPING,true
-bread,2019-10-15,(2) SHOPPING,false
+cheese,2020-12-12,SHOPPING,true
```
Comment (kazaaz) [inline src/resources/task.csv:1]: Just a tip: If you like to ignore some files in git, such that they are not pushed to github or versioned. Then it's a good idea to add them to  `.gitignore`: https://www.atlassian.com/git/tutorials/saving-changes/gitignore

File: src/App.java [hunk 2]
```diff
@@ -104,10 +104,11 @@ private void collectDataToCreateTask() {
...
-        String[] projects = {"(1) ERRANDS", "(2) SHOPPING", "(3) APPOINTMENTS", "(4) OTHER"};
+        String[] projects = {"ERRANDS", "SHOPPING", "APPOINTMENTS", "OTHER"};
             for (int i = 0; i<projects.length; i++){
-            System.out.println(projects[i]);
+            System.out.println("(" + (i+1) + ") " + projects[i]);
             }
+        System.out.println("Or enter another name for your project");
         project = nextStep();
 
         switch (project) {
```
Comment (kazaaz) [inline src/App.java:140]: If you like you can use `scanner.hasNextInt()` to check if the user chosen an existing project or a predefined one. In that way you avoid duplication of switch cases and you won't have to add a new switch case for every new added project.

File: src/ToDoList.java [hunk 3]
```diff
@@ -51,50 +51,112 @@ public void sortListByProject(){
...
+    public int sizeOfTheList(){
+        return myTasks.size();
+    }
+
+    /**
+     * Display a task chosen by the user for future manipulations
+     * @param index
+     * @return String task
+     */
+    public String getTheTaskFromTheList(int index){
```
Comment (kazaaz) [inline src/ToDoList.java]: Any specific reason for returning the task as `String` instead of as `Task`?

**Review by kazaaz — COMMENTED** [review]
:+1:

File: src/resources/task.csv (hunk 1 above)
Comment (krismutalapova) [reply src/resources/task.csv:1]: Thanks, I have .gitignore I just forgot to add a csv file into it

**Pull request merged**